src/app/components/split-view/index.html

```diff
@@ -0,0 +1,239 @@
+<sky-tabbed-demo-page
+  pageTitle="Split view"
+>
+  <sky-demo-page-summary>
+    <p>
+      The split view component displays a list alongside a workspace where users can view details for the selected item and take actions.
+    </p>
+    <sky-alert alertType="warning">
+      This component is an initial release candidate.
```

Comment:
Is this warning too vague? Should we mention that, though not anticipated, changes to the component and its implementation detail may occur prior to official release? @Blackbaud-AlexKingman thoughts?

Reply:
We can flesh out the statement if we want, but when Alex and I discussed this, we thought that Steve had been against calling out the possibility of changes since it's not supposed to change. But then on the Slack thread the other day, he seemed agreeable to fleshing this out. He also said to remove "initial."

So what do you guys think of fleshing this out to: "This component is a release candidate. We do not plan to change the component or its implementation details before the official release, but such changes are a possibility."

Reply:
Second sentence maybe "We do not plan to make any breaking changes to the component before the official release..."?

If we don't want to expand the text here, we should switch from an Alert to a Label saying "Release candidate"

Reply:
I like the updated text, so I can make that change. But I'm good with a "Release candidate" label as well. Which do you prefer?

Reply:
I updated the text. (Let me know if we should switch to a label.)